package opensolid.serialized;

message Value {
    optional int64 int_value = 1;
    optional double double_value = 2;
    optional string string_value = 3;
    optional List list_value = 4;
}

message List {
    repeated Value item = 1;
}
